step.yml

```diff
@@ -211,7 +218,7 @@ inputs:
         
         The value needs to be a newline separated list of colon separated items with the following structure: 
         
-        *<path>:<env_key>*
+        **&lt;path>:&lt;env_key>**
         
         The `path` is the path to the file or folder and the `env_key` is the name of the environment variable to which this value is associated.
```

Comment:
I'd recommend changing this to:
The `path` is the path to the file or folder. The `env_key` is the name of the environment variable with which this value is associated.